src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -51,7 +51,7 @@ namespace System.Management.Automation
     ///     - There is no ITypeDescriptorContext.
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-macos`
Language Primitive Tests.Test convertion with .Net Core intrinsic type convertor
```powershell
Expected strings to be the same, but they were different.
Expected length: 20
Actual length:   21
Strings differ at index 20.
Expected: 'http://test.site.com'
But was:  'http://test.site.com/'
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 44
44:         $result.AbsoluteUri | Should -Be 'http://test.site.com'
```
New-ModuleManifest tests.Relative URIs are not allowed
```powershell
Cannot convert value "../foo" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Module/NewModuleManifest.Tests.ps1: line 115
115:         $testUri = [Uri]"../foo"

```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Configuration.ConfigurationManager.dll; SideIndicator==>}.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```
ConvertTo-Html Tests.Test ConvertTo-Html using page parameters
```powershell
Cannot bind parameter 'CssUri'. Cannot convert value "page.css" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1: line 83
83:         $returnString = ($customObject | ConvertTo-Html -Title "Custom Object" -Body "Body Text" -CssUri "page.css" -As List) -join $newLine

```